docs/fromConfig/list.md

```diff
@@ -0,0 +1,18 @@
+---
+id: list
```

Comment:
I _think_ you have to add this somewhere for it to show in the sidebar, probably inside website/page.js or something like that